.circleci/config.yml

```diff
@@ -19,8 +19,14 @@ jobs:
           paths:
             - ~/yarn_cache
       - run:
-          working_directory: ~/expo-dev-tools/packages/dev-tools
+          name: 'dev-tools tests'
+          working_directory: ~/expo-cli/packages/dev-tools
           command: yarn test
       - run:
-          working_directory: ~/expo-dev-tools/packages/expo-codemod
+          name: 'json-file tests'
+          working_directory: ~/expo-cli/packages/json-file
+          command: yarn test
```

Comment:
do you think it would be useful to add `yarn build` to check for type errors too? (the babel preset just converts TS but doesn't do typechecking)

Reply:
This already happens as a part of the [bootstrap script](https://github.com/expo/expo-cli/blob/4079500ae3edcec5b153013aec2ee3841b399173/package.json#L7-L8) because all our packages include build as the `prepare` script which we run post bootstrap.